views/partials/tips.hbs

```diff
@@ -1,4 +1,175 @@
+
 {{!< default }}
+
+<div id="password-advice" class="section-container">
+  <div class="section-wrapper">
+  <h3>Passwords</h3>
+  <h5>The Key to Account Security</h5>
+  <p>Your passwords protect all of the personal information that resides in your online accounts, from bank balances and credit card numbers to your home address and private messages. Passwords are so valuable that thousands of passwords are stolen every day and accounts are traded on the black market.  Take these six steps to protect yours.</p>
+  </div>
+</div>
+<div id="tips-container" class="section-container">
+    <div class="tips-wrapper">
+      <div class="tip-container">
+          <div class="tip">
+              <div class="svg-background">
+                <img src="img/Password_Tip_1@2x.png" alt="Account Security Tip #1">
```

Comment:
Should we delete the .svg files now? I think if we're switching to PNG images, they're potentially orphaned:

```diff
- <img src="img/Tip1.svg">
+ <img src="img/Password_Tip_1@2x.png" alt="Account Security Tip #1">
```